config/FindVIGRANUMPY_DEPENDENCIES.cmake

```diff
@@ -68,10 +68,10 @@ IF(PYTHONINTERP_FOUND)
             execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
                              "from distutils.sysconfig import *; print(get_config_var('LDLIBRARY'))"
                               OUTPUT_VARIABLE PYTHON_LIBRARY_NAME OUTPUT_STRIP_TRAILING_WHITESPACE)
-            execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
-                             "from distutils.sysconfig import *; print(get_config_var('LIBDIR'))"
-        			           OUTPUT_VARIABLE PYTHON_LIBRARY_PREFIX OUTPUT_STRIP_TRAILING_WHITESPACE)
         ENDIF()
+        execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
+                        "from distutils.sysconfig import *; print(get_config_var('LIBDIR'))"
+                        OUTPUT_VARIABLE PYTHON_LIBRARY_PREFIX OUTPUT_STRIP_TRAILING_WHITESPACE)
         FIND_LIBRARY(PYTHON_LIBRARIES ${PYTHON_LIBRARY_NAME} HINTS "${PYTHON_LIBRARY_PREFIX}" "${PYTHON_PREFIX}"
```

Comment:
Doing this on Windows has not effect since `get_config_var('LIBDIR')` returns `None`. However, the Windows build works for me, so don't worry about it. (The AppVeyor failure was do to `FIND_PACKAGE(PythonInterp ${PYTHON_VERSION})` being called with empty PYTHON_VERSION variable. I simply had to remove the quotes.)